patch/kconfig-inclusions

```diff
@@ -211,7 +180,6 @@ CONFIG_UIO_PENMSI=m
 CONFIG_ARM64_VA_BITS_39=y
 CONFIG_COMMON_CLK_CS2000_CP=y
 CONFIG_I2C_RD1173=y
-CONFIG_PENSANDO_SOC_PENFW=y
```

Comment:
I don't think this should be removed?

Reply:
The tool uses configParser class which removes duplicate configs. CONFIG_PENSANDO_SOC_PENFW, CONFIG_GPIOLIB & CONFIG_OF which are duplicates.

If you insist however, i can make sure the nothing is touched outside of mellanox boundaries

Reply:
Ah, I didn't realize it's a duplicate; that's fine then.